pokedex_async_redux/lib/features/pokemon_details/pokemon_details_connector.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:flutter/material.dart';
+import 'package:pokedex_asyn_redux/api/model/pokemon.dart';
+import 'package:pokedex_asyn_redux/features/pokemon_details/pokemon_details_page.dart';
+import 'package:pokedex_asyn_redux/features/pokemon_details/pokemon_details_vm.dart';
+import 'package:pokedex_asyn_redux/state/action/pokemon_actions.dart';
+import 'package:pokedex_asyn_redux/state/app_state.dart';
+
+class PokemonDetailsConnector extends StatelessWidget {
+  final String pokemonName;
```

Comment:
seems these properties: pokemonName & pokemonImageUrl, aren't needed anymore since you can already utilize what's inside the `pokemon` object 🙂